sympy/stats/stochastic_process_types.py

```diff
@@ -413,7 +414,7 @@ def _check_trans_probs(self, trans_probs, row_sum=1):
         if not isinstance(trans_probs, MatrixSymbol):
             rows = trans_probs.tolist()
             for row in rows:
-                if (sum(row) - row_sum) != 0:
+                if sum(row) - row_sum != 0:
```

Comment:
```suggestion
                if (sum(row) - row_sum) != 0:
```
Doesn't really matter much, irrelevant to the motive of this PR.